### ISSUE TITLE
Persons: Display alternate side menus for person and institutions

### ISSUE THREAD

**Issue by mMoliere** [majlis-erc/manuForma#640]
@wsalesky Users can choose between two templates for this entity type: natural persons and institutions. This should also be reflected in the side menues. At the moment, manuForma displays the menus for both variants which is not appropriate.

This example is from 35.xml

![Screenshot 2024-08-27 at 16-03-23 Edit Person TEI records](https://github.com/user-attachments/assets/3a5b3f37-ca12-427d-ab3f-cd109b52a6ad)

"Heritage Data" is part of both entry variants, so it should be displayed regardless.

From a TEI perspective, the major difference is that the `listPerson` element contains an element `org` instead of `person`. (This is allowed by the TEI guidelines.) Therefore a possible switch in manuForma could check if the element `org` is a childelement of `listPerson` and then display the institution side menus. Otherwise, display the person side menues.

**Comment by mMoliere** [comment]
@wsalesky Do you think this is possible?

**Comment by wsalesky** [comment]
@mMoliere what are the differences in the TEI that I could use to base the different menus on?

**Comment by mMoliere** [comment]
@wsalesky 

Person has the following structure:

body
-----listPerson
----------person

Organisation is similar but has org instead of person:

body
-----listPerson
----------**org**

**Comment by wsalesky** [comment]
I will see what we can doOn Dec 10, 2024, at 2:23 PM, mMoliere ***@***.***> wrote:﻿
@wsalesky
Person has the following structure:
body
-----listPerson
----------person
Organisation is similar but has org instead of person:
body
-----listPerson
----------org

**Comment by wsalesky** [comment]
@mMoliere I don't think this can be done in a single form, you could make them into 2 different forms. Would that work?

**Comment by mMoliere** [comment]
@wsalesky I will ask.

**Comment by mMoliere** [comment]
@wsalesky We will stick to what we have now. Is there a chance to insert a horizontal line between the two blocks?
